Dockerfile

```diff
@@ -0,0 +1,9 @@
+FROM node:latest
+
+WORKDIR /
+
+COPY . .
+
+RUN npm install 
+
+CMD cat overlay.yaml | node bin.js
```

Comment:
We can set the command to be `bin.js`, that way it's possible to make use of stdin and we may not need to mount files. Although mounting files may prove useful later on.

Reply:
This was more of a suggestion on how to do it. I'm happy with any way you think would work the best. For example when we use this we are running command:
```
docker run --rm -v=${PWD}/overlay.yaml:/overlay.yaml -v=${PWD}/test.yaml:/test.yaml jasmingacic/overlay-cli
```

We can adjust to run it any other way. 

And  you are absolutely right about the docker image I used here. Current size is about 140MB. I'm not well versed with Node so I took the first thing I found. 

Do you want me to add those changes?